src/searchbar/renderer/_DateOrTextInput.stories.tsx

```diff
@@ -0,0 +1,99 @@
+import DateFnsUtils from "@date-io/date-fns";
+import { action } from "@storybook/addon-actions";
+import MuiPickersUtilsProvider from "material-ui-pickers/MuiPickersUtilsProvider";
+import React from "react";
+import DateOrTextInput, { forcePickerStateOpen } from "./_DateOrTextInput";
+
+export default {
+    title: "searchbar/renderer/DateOrTextInput",
+    component: DateOrTextInput,
+};
+
+function Wrapper({ children }) {
+    return <MuiPickersUtilsProvider utils={DateFnsUtils}>
+        {children}
+    </MuiPickersUtilsProvider>;
+}
+
+const date = new Date("2020-05-08 10:22:40");
+
+export const dateOnly = () => <Wrapper>
```

Comment:
The change **value** for 'date only' pickers also include a timestamp. 
This would sometimes be the ts of the user clicked on the datepicker.
![image](https://user-images.githubusercontent.com/34885540/112822412-33909f00-9088-11eb-8e5d-b33866347b45.png)

Shouldn't we hardcode the hour to be 00:00 ? or does the hour get ignored when creating a search query in finder?